src/main/kotlin/co/netsoc/netsocbot/App.kt

```diff
@@ -8,19 +8,27 @@ import co.netsoc.netsocbot.commands.registerDMs
 import com.jessecorbett.diskord.util.*
 import co.netsoc.netsocbot.utils.isDM
 import java.net.UnknownHostException
+import java.io.File
 
 val BOT_TOKEN: String = System.getenv("NETSOCBOT_TOKEN")
 val PREFIX = System.getenv("NETSOCBOT_PREFIX") ?: "!"
 val ROLEIDS = (System.getenv("NETSOCBOT_ROLEIDS") ?: "").split(",")
 
+
 suspend fun setup() {
     val environmentVariables = arrayOf("NETSOCBOT_TOKEN", "NETSOCBOT_ROLEIDS", "NETSOCBOT_SENDGRID_TOKEN")
     for (variable in environmentVariables) {
+        println(variable)
         if (System.getenv(variable) == null) {
             println("$variable not set\nExiting")
             exitProcess(1)
         }
     }
+    var fileObject = File("brownieLeaderboard.txt")
```

Comment:
I would use a dB instead of a file, simply a pair of user ID to amount of brownie points should be fine